roles/flent/templates/server.yml.j2

```diff
@@ -0,0 +1,38 @@
+---
+kind: Pod
```

Comment:
Metadata collections fails, we need to convert this resource to a Job to fix YAML indenting issues as you already got in the workload pod.